Source/Urho3D/Core/Profiler.h

```diff
@@ -22,9 +22,9 @@
 
 #pragma once
 
-#include <tracy/Tracy.hpp>
+#include <tracy/public/tracy/Tracy.hpp>
```

Comment:
This is probably not necessary and can be fixed in CMakeLists.txt by providing a proper include path.

Reply:
I just want to keep the directory structure style with upstream branch.

Reply:
You can keep directory structure. You need to modify `target_include_directories` in tracy `CMakeLists.txt` to include "tracy/public".